_events/sr2019/competition.md

```diff
@@ -3,24 +3,50 @@ title: Competition
 date: 2019-04-06 9:00:00
 layout: event
 type: competition
-location: Southampton University Highfield Campus
+location: Southampton University
 ---
 
 The Student Robotics Competition marks the ultimate stage of SR2019, showcasing the months of preparation. Our teams will work hard to push their robots to the limit and quickly make repairs between league matches. We will finish the weekend with the finals where the best bots will be named victorious.
 
-## Directions
+## Schedule
 
-The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. 
+The competition will be held from *Saturday 6th April* until *Sunday 7th April*. The exact start/end times are to be confirmed, but doors open 9am and we aim to finish around 6pm.
```

Comment:
Saying that the times are yet to be confirmed reads really oddly given that we then provide some actual times. Can we do one or the other?

Reply:
```suggestion
The competition will be held from *Saturday 6th April* until *Sunday 7th April*. Doors open 9am and we aim to finish around 6pm.
```

Reply:
https://github.com/srobo/website/pull/132/commits/960d3d4d6da64da341c148fbaba151747e2df1e2